pkg/register/restResolver_test.go

```diff
@@ -64,7 +65,7 @@ func Test_Resolver_Successful_Get(t *testing.T) {
 	httpmock.RegisterResponder("GET", addr.String()+"/1.0/discover/"+url.QueryEscape(ValidID), responder)
 
 	// Place the call
-	_, err := rslv.GetDocument(ValidID)
+	_, err := rslv.GetDocument(context.TODO(), ValidID)
```

Comment:
It would have been nice to check that the context is passed through but it appears that `httpmock` has no support for this.
(In another services we use a `gomock` of a subset of `HttpClient` - but that would mean changing all of these so I think it's not worth it for now.)